Formula/d/dug.rb

```diff
@@ -1,10 +1,9 @@
 class Dug < Formula
   desc "Global DNS propagation checker that gives pretty output"
   homepage "https://dug.unfrl.com"
-  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.94.tar.gz"
-  sha256 "f97952be49d93ed66f1cc7e40bf7004928e6573077839a18f5be371c80e2c16b"
+  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.98.tar.gz"
```

Comment:
I assume this should be `https://git.unfrl.com/Unfrl/dug/archive/0.0.100.tar.gz` but not sure what to do until the [dotnet formulae](https://formulae.brew.sh/formula/dotnet) supports version 10 :thinking: